internal/convert.go

```diff
@@ -168,27 +168,52 @@ func (conv *Conv) SetDataMode() {
 	conv.mode = dataOnly
 }
 
+func dequeue(queue []string) []string {
+	return queue[1:]
+}
+
 // GetDDL Schema returns the Spanner schema that has been constructed so far.
-// Return DDL in alphabetical table order.
+// We sort DDL in alphabetical order, but we maintain the order of interleaved
+// relation by passing tables through a queue
 func (conv *Conv) GetDDL(c ddl.Config) []string {
```

Comment:
We should have a test of this new functionality -- in convert_test.go?
Maybe extend existing TestGetDDL function?

Reply:
added tests for new functionality